src/System.Threading.AccessControl/src/Interop/Interop.Manual.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+internal static partial class Interop
+{
+    internal static partial class mincore
```

Comment:
This looks like it overlaps with constants already in Common if it does please merge. If it doesn't we should probably put it into the common interop.